Doc/library/email.rst

```diff
@@ -16,7 +16,7 @@ The :mod:`email` package is a library for managing email messages.  It is
 specifically *not* designed to do any sending of email messages to SMTP
 (:rfc:`2821`), NNTP, or other servers; those are functions of modules such as
 :mod:`smtplib` and :mod:`nntplib`.  The :mod:`email` package attempts to be as
-RFC-compliant as possible, supporting :rfc:`5233` and :rfc:`6532`, as well as
+RFC-compliant as possible, supporting :rfc:`5232` and :rfc:`6532`, as well as
```

Comment:
```suggestion
RFC-compliant as possible, supporting :rfc:`5322` and :rfc:`6532`, as well as
```

Reply:
Thanks for catching my typo!

I fixed `email.compat32-message.rst` as well now.